### ISSUE TITLE
[Acceptance] Nightly failure in `apiUserLDAP/sharingLocalUserLdapUser.feature:136`

### ISSUE THREAD

**Issue by grgprarup** [owncloud/user_ldap#809]
The acceptance test is failing in scenario `apiUserLDAP/sharingLocalUserLdapUser.feature:136` with `daily master` server
```feature
@issue-364
  Scenario: Share a folder from an LDAP user to a local user                                                                      # /var/www/owncloud/testrunner/apps/user_ldap/tests/acceptance/features/apiUserLDAP/sharingLocalUserLdapUser.feature:136
    Given user "Alice" has shared folder "/PARENT" with user "local-user"                                                         # FeatureContext::userHasSharedFileWithUserUsingTheSharingApi()
    When the administrator sets the LDAP config "LDAPTestId" key "ldapHost" to "not-existing" using the occ command               # UserLdapGeneralContext::ldapConfigHasKeySetTo()
    And user "local-user" downloads file "/PARENT (2)/parent.txt" using the WebDAV API                                            # FeatureContext::userDownloadsFileUsingTheAPI()
    Then the HTTP status code should be "500"                                                                                     # FeatureContext::thenTheHTTPStatusCodeShouldBe()
      HTTP status code 404 is not the expected value 500
      Failed asserting that 404 matches expected '500'.
    And the content of file "/PARENT (2)/parent.txt" for user "local-user" should be:                                             # FeatureContext::contentOfFileForUserShouldBePyString()
      """
      <?xml version="1.0" encoding="utf-8"?>
      <d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
        <s:exception>InvalidArgumentException</s:exception>
        <s:message>Jail rootPath is null</s:message>
      </d:error>
      
      """
    When user "local-user" gets the properties of file "/PARENT (2)/parent.txt" using the WebDAV API                              # WebDavPropertiesContext::userGetsThePropertiesOfFolder()
    Then the HTTP status code should be "500"                                                                                     # FeatureContext::thenTheHTTPStatusCodeShouldBe()
    When the administrator sets the LDAP config "LDAPTestId" key "ldapHost" to "%ldap_host_without_scheme%" using the occ command # UserLdapGeneralContext::ldapConfigHasKeySetTo()
    And user "local-user" gets the properties of file "/PARENT (2)/parent.txt" using the WebDAV API                               # WebDavPropertiesContext::userGetsThePropertiesOfFolder()
    Then the properties response should contain an etag                                                                           # WebDavPropertiesContext::thePropertiesResponseShouldContainAnEtag()
    And the content of file "/PARENT (2)/parent.txt" for user "local-user" should be:                                             # FeatureContext::contentOfFileForUserShouldBePyString()
      """
      ownCloud test text file parent
      
      """
```
Builds:
https://drone.owncloud.com/owncloud/user_ldap/4748/28/12

**Comment by saw-jan** [comment]
Could be similar to https://github.com/owncloud/files_lifecycle/issues/543
and might be due to changes in https://github.com/owncloud/core/commit/7706791af23629041ec12bbc2c9e528c54dfdb76

**Comment by PrajwolAmatya** [comment]
Currently the failure with status code `404` is occurring only in the `apiUserLDAP-master` pipelines but on the `apiUserLDAP-latest` pipelines the test passes with status code `500`.

**Comment by phil-davis** [comment]
> Currently the failure with status code `404` is occurring only in the `apiUserLDAP-master` pipelines but on the `apiUserLDAP-latest` pipelines the test passes with status code `500`.

which means that the problem that caused the 500 is fixed in master, which is good.
you might need to skip the scenario on Oc 10.13.0, 10.13.1 (and the fixed code looks like it is in 10.13.2 release, which should soon be "latest")

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/user_ldap/4748/31/5
The pipeline for "latest" that passed shows:
`Fetched 10.13.1 - channel: stable - build at 2023-09-06T16:24:36+00:00 - SHA: d06b1d870377662dd9ff327485acc2d3e1991bcb`

As at last night, "latest" was fetching 10.13.1.
Downloading "latest" now still gives me 10.13.1 - but that will switch over some time soon.
